### PR TITLE
Convert swarm to newer 2822-based format

### PR DESCRIPTION
**Pull request by tianon**

### CODE REVIEW

**Comment by yosifkit** [comment]
cc @dongluochen and @nishanttotla
Hopefully this will make it easier to update :smile:

**Comment by nishanttotla** [comment]
LGTM :)

File: library/swarm [hunk 1]
```diff
@@ -1,10 +1,9 @@
...
+             Nishant Totla <nishanttotla@gmail.com> (@nishanttotla),
+             Victor Vieux <vieux@docker.com> (@vieux)
+GitRepo: https://github.com/docker/swarm-library-image.git
 
-
-1.2.4: git://github.com/docker/swarm-library-image@ad016670d877529ab32e7ef2a1323777a06cb46a
-1.2.5: git://github.com/docker/swarm-library-image@67e6cf4171fcabd2e3b8b809105e7dfb15114ce1
-latest: git://github.com/docker/swarm-library-image@67e6cf4171fcabd2e3b8b809105e7dfb15114ce1
+Tags: 1.2.5, latest
+GitCommit: 67e6cf4171fcabd2e3b8b809105e7dfb15114ce1
```
Comment (dongluochen) [inline library/swarm]: If we add a `1.2.6-rc1`, what should it look like?

Comment (tianon) [inline library/swarm]: Just add a new section; something like this should do it:
```
Maintainers: ...
GitRepo: ...

Tags: 1.2.5, latest
GitCommit: 67e6cf4171fcabd2e3b8b809105e7dfb15114ce1

Tags: 1.2.6-rc1
GitCommit: xxxxxxx
```

Then, once 1.2.6 officially releases, you remove the second entry and update the first again. :+1:

Comment (dongluochen) [inline library/swarm]: Sounds good.

**Comment by dongluochen** [comment]
LGTM

**Pull request merged**